go/vt/wrangler/traffic_switcher.go

```diff
@@ -581,6 +581,13 @@ func (wr *Wrangler) SwitchWrites(ctx context.Context, targetKeyspace, workflowNa
 			return 0, nil, err
 		}
 
+		ts.Logger().Infof("Resetting sequences")
+		if err := sw.resetSequences(ctx); err != nil {
+			ts.Logger().Errorf("resetSequences failed: %v", err)
```

Comment:
can we not call directly into `ts.resetSequences` not sure why redirection is needed.
I understand that other methods are also written in the same way.
Just calling it out.

Reply:
We invoke an interface here, so that the same code path is used for dry runs and normal execution. When dry run is invoked `sw.resetSequences()` will just log the action to the user and not call `ts.resetSequences()`.